test/unit/helpers/xhr/customFactor_error.js

```diff
@@ -0,0 +1,13 @@
+define({
```

Comment:
can we rename this file to `CUSTOM_FACTOR_error`?
And also, do we need this file? it seems to be a generic error, so we might have already another file that we can use

Reply:
This error is in specific files like SMS verify and resend.. Maybe we can use RECOVERY_error.js? If not, i'll rename this file to something more generic